app/models/audiovisual_work.rb

```diff
@@ -95,8 +95,8 @@ class AudiovisualWork < ActiveFedora::Base
 
   # This property will put together type and note content in a parsable string
   # Also covers statement of responsibility
-  property :note, predicate: ::RDF::Vocab::SKOS.note do |index|
-    index.as :stored_searchable
+  property :note, predicate: ::RDF::Vocab::SKOS.note, multiple: false do |index|
```

Comment:
@jenyoung @jlhardes 
There are two options here: a single RDF triple for SKOS:note with a JSON array of objects:
```
<http://127.0.0.1:8984/rest/dev/ws/85/9f/64/ws859f64s> <http://www.w3.org/2004/02/skos/core#note> "[\n  {'type': 'Statement of Responsibility', 'note_body': 'Jane Doe'},\n  {'type': 'Statement of Responsibility', 'note_body': 'John Doe'}\n]" .
```
or multiple triples with one for each object:
```
<http://127.0.0.1:8984/rest/dev/ws/85/9f/64/ws859f64s> <http://www.w3.org/2004/02/skos/core#note> "{'type': 'Statement of Responsibility', 'note_body': 'Jane Doe'}" .
<http://127.0.0.1:8984/rest/dev/ws/85/9f/64/ws859f64s> <http://www.w3.org/2004/02/skos/core#note> "{'type': 'Statement of Responsibility', 'note_body': 'John Doe'}" .
```
Which one makes the most sense to you two?  I'm leaning towards the latter.

Reply:
We decided to go with the former so we could ensure ordering.